README.md

```diff
@@ -35,70 +42,6 @@ Maintainers ([@open-telemetry/ruby-maintainers](https://github.com/orgs/open-tel
 
 *Find more about the maintainer role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).*
 
-## Installation
```

Comment:
Do we want to move this info outside of the repo readme?  I know it's not great have docs in two places as they are almost guaranteed to diverge but I feel strongly towards to keeping something like this in the readme.  It's really handy when the simple getting started bit is inside the codebase.  We could push anything beyond the basic "getting started" into the microsite, as it has better support for structuring documentation.

WDYT?